pom.xml

```diff
@@ -28,18 +28,34 @@
         <selenium.version>3.1.0</selenium.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
```

Comment:
Thanks - somehow I've never encountered a BOM (just put entries directly in <dependencyManagement>)